### ISSUE TITLE
Wallet merging of copies

### ISSUE THREAD

**Issue by Vatavuk** [zold-io/java-api#70]
According to the whitepaper merging of two wallets is an operation of merging `copies` into a local one.
Merge algorithm first sorts the `copies` and then merges transactions from that copies.
Currently, sorting is done outside `Wallet.merge()` method. It is part of `Remote.pull()` method.
I propose to add an additional method to `Wallet` : `Wallet merge(Copies copies)` and remove explicit sorting from `Remote.pull()`.
Similar to what we have in [ruby version](https://github.com/zold-io/zold/blob/7f30678b207c53afebc264cce40d54244e61ce91/lib/zold/commands/merge.rb#L69)

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@8ff2a38c-103c-4d3f-89c6-ae548aaac564, thread: pool-219-thread-1
com.zerocracy.radars.github.ThrottledComments[74] java.lang.IllegalStateException: Can't post an identical message to zold-io/java-api#70: @llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue
```

[0.29.8](https://github.com/zerocracy/farm): Issue: zold-io/java-api#70, Action: `opened`

**Comment by llorllale** [comment]
@vatavuk agreed. I also think `Copy` should be a `Wallet` with the existing, extra attribute `score()`.

**Comment by 0crat** [comment]
Job [#70](https://github.com/zold-io/java-api/issues/70) is now in scope, role is `DEV`

**Comment by llorllale** [comment]
@0crat in

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @vatavuk[/z](https://www.0crat.com/u/vatavuk)

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/zold-io/java-api/issues/70#issuecomment-412345513))

@llorllale Job [#70](https://github.com/zold-io/java-api/issues/70) is already in scope
